libraries/libfc/include/fc/bitutil.hpp

```diff
@@ -33,6 +33,19 @@ inline uint32_t endian_reverse_u32( uint32_t x )
 }
 
 // Using uint8_t boost::dynamic_bitset provides a more expected raw pack/unpack format
-using dynamic_bitset = boost::dynamic_bitset<uint8_t>;
+struct dynamic_bitset : public boost::dynamic_bitset<uint8_t> {
+   using base = boost::dynamic_bitset<uint8_t>;
+
+   // stream output function overriding the one in `boost::dynamic_bitset`. Currently it outputs
+   // the same as `boost::dynamic_bitset`.
+   // ------------------------------------------------------------------------------------------
+   template <typename Stream>
+   friend Stream& operator<<(Stream& ds, const dynamic_bitset& bs) {
+      ds << static_cast<base>(bs);
+      return ds;
+   }
+
+   using base::base;
+};
```

Comment:
What is the point of this?

Reply:
There was more of a point when the serialization was changed to have a "0b" prefix.
I thought it was better to leave it so that the serialization is explicitly defaulted to the `boost::dynamic_bitset<uint8_t>` instead of happening in the background.

Reply:
I think it would be better to remove it. It is just noise. We probably should add a test that verifies expected serialization so `boost::dynamic_bitset` doesn't change it out from under us.

Reply:
Or I can add my `bitset.hpp` from cdt and use that instead of `boost::dynamic_bitset`, it would be more consistent.

Reply:
Sounds like an excellent idea to me.